examples/bind.amp.html

```diff
@@ -35,10 +35,8 @@
   <p [class]="textClass">This text will have have a red background color</p>
   <amp-img src="https://ampbyexample.com/img/Border_Collie.jpg" [src]="imgSrc" width=100 [width]="imgSize" height=100 [height]="imgSize" alt="asdf" [alt]="imgAlt"></amp-img>
   <p>The image above will increase in size and change its src</p>
-  <!--
-  <amp-video src="https://ampbyexample.com/video/tokyo.mp4" [src]="videoSrc" width=480 height=270 controls></amp-video>
-  -->
+  <amp-video src="https://ampbyexample.com/video/tokyo.mp4" [src]="videoSrc" width=480 height=270 autoplay [controls]="videoControls"></amp-video>
```

Comment:
remove autoplay given the the explanation on https://github.com/ampproject/amphtml/pull/6905#issuecomment-270780481